cfe_internal/enterprise/templates/httpd.conf.mustache

```diff
@@ -215,7 +215,7 @@ LogLevel warn
     object-src 'none'; \
     frame-src 'self'; \
     child-src 'self'; \
-    img-src 'self' data: blob: avatars.githubusercontent.com badges.gitter.im fonts.gstatic.com kiwiirc.com raw.githubusercontent.com; \
+    img-src 'self' data: blob: avatars.githubusercontent.com badges.gitter.im fonts.gstatic.com kiwiirc.com raw.githubusercontent.com raw.github.com; \
```

Comment:
needed here as well right? https://github.com/cfengine/buildscripts/blob/master/deps-packaging/apache/httpd.conf#L210

Reply:
Yep, indeed. here it is. https://github.com/cfengine/buildscripts/pull/1562